libs/execution_context/ebpf_maps.c

```diff
@@ -496,20 +496,12 @@ _check_value_type(_In_ const ebpf_core_map_t* outer_map, _In_ const ebpf_core_ob
     return allowed;
 }
 
-// Validate that a value handle is appropriate for this map,
-// and if so, return a pointer to the object with that handle.
+// Validate that a value object is appropriate for this map.
 static ebpf_result_t
-_get_map_value_object(
-    _In_ const ebpf_core_map_t* map,
-    ebpf_handle_t value_handle,
-    ebpf_object_type_t value_type,
-    _Outptr_ ebpf_core_object_t** value_object_result)
+_validate_map_value_object(
```

Comment:
if this function requires the object lock to be held, please add required SAL annotations.

Reply:
Added.